packages/remix-cloudflare-workers/worker.ts

```diff
@@ -10,14 +10,20 @@ import {
 import type { AppLoadContext, ServerBuild } from "@remix-run/cloudflare";
 import { createRequestHandler as createRemixRequestHandler } from "@remix-run/cloudflare";
 
+export type Evt<Env = unknown> = {
+  request: Request
+  waitUntil: (promise: Promise<any>) => void
+  env: Env,
+}
+
 /**
  * A function that returns the value to use as `context` in route `loader` and
  * `action` functions.
  *
  * You can think of this as an escape hatch that allows you to pass
  * environment/platform-specific values through to your loader/action.
  */
-export type GetLoadContextFunction = (event: FetchEvent) => AppLoadContext;
+export type GetLoadContextFunction = (event: FetchEvent | Evt) => AppLoadContext;
```

Comment:
I think it would make sense to parameterise `<Environment>` for this & dependent functions, to allow the user to easily type it. So:

```ts
export type GetLoadContextFunction<Environment = unknown> = (event: FetchEvent | Evt<Environment>) => AppLoadContext;
```